src/index.ts

```diff
@@ -68,11 +67,19 @@ app.get(`/:id`, (request, response) => {
 app.post(`/`, (request, response) => {
     // Check if it is a valid url
     const { url } = request.body;
-    if (!url || !config.urlRegex.test(url)) {
+    
+    try {
+        url = new URL(string);
+    } catch (_) {
```

Comment:
Can be replaced with `} catch {` or `} catch () {`, not sure which one it was again. Just see for which one VSCode throws an error.